test/test.hs

```diff
@@ -291,6 +295,28 @@ testHoughLinesP = do
     lineSegments <- houghLinesP 1 (pi / 180) 100 Nothing Nothing edgeImgM
     assertBool "no lines found" (V.length lineSegments > 0)
 
+testNewCascadeClassifierAlgorithm :: HU.Assertion
+testNewCascadeClassifierAlgorithm = do
+  mbCC <- newCascadeClassifier "/this/is/bogus.xml"
+  case mbCC of
+    Nothing -> return ()
+    Just _cc -> fail "expected Nothing from newCascadeClassifier"
+
+testCascadeClassifierDetectMultiScaleArnold :: HU.Assertion
+testCascadeClassifierDetectMultiScaleArnold = do
+  Just ccFrontal <- newCascadeClassifier "/usr/local/share/OpenCV/haarcascades/haarcascade_frontalface_default.xml"
```

Comment:
I checked the license for the `xml` file. It is the BSD 3-Clause License with Intel Corporation as the copyright holder. Is see no problems with including it in this repository. Using it only in the test-suite and examples should free users from the responsibility of reproducing the license in binary redistributions.